app.js

```diff
@@ -345,6 +345,8 @@ app.post('/project/:id/request', projects);
 app.delete('/project/:id', projects);
 app.put('/project/:id/request', projects);
 app.post('/ask/project', projects);
+app.get('/pending', projects);
```

Comment:
New routes

Reply:
wouldbe better to get "named" routes:

/pending/project , /pending/project/:id

best to handle later other pending stuff